src/main/java/com/minecolonies/coremod/network/messages/server/BuildToolPlaceMessage.java

```diff
@@ -206,7 +206,7 @@ private static void handleHut(
         }
 
         final ItemStack stack = new ItemStack(state.getBlock(), 1);
-        final Block block = stack.getItem() instanceof BlockItem ? ((BlockItem) stack.getItem()).getBlock() : null;
+        final Block block = state.getBlock();
```

Comment:
I think it would be more logical to swap those lines
```java
final Block block = state.getBlock()
final ItemStack stack = new ItemStack(block, 1);
```